### PR TITLE
Only show 5 next/prev pages

### PR DESCRIPTION
**Pull request by sethvargo**
10 rolls off the page when there's a lot of pages.

**Release Note**

```release-note
Only show 5 next/prev pages in pagination.
```

### CODE REVIEW

**Review by whaught — APPROVED** [review]

File: pkg/database/pagination.go [hunk 1]
```diff
@@ -66,13 +66,13 @@ func PaginateFn(query *gorm.DB, page, limit uint64, populateFn func(query *gorm.
 		remaining := total - current
 		nextPages = uint64(math.Ceil(float64(remaining) / float64(limit)))
 		if nextPages > 10 {
```
Comment (whaught) [inline pkg/database/pagination.go:68]: a `const` might be good

does this need to be `nextPages > 5`

**Comment by sethvargo** [comment]
@whaught good call on constants and that check. Fixed. PTAnotherL.

**Review by whaught — APPROVED** [review]

File: pkg/database/pagination.go [hunk 2]
```diff
@@ -14,6 +14,12 @@
 
 package database
 
+const (
```
Comment (whaught) [inline pkg/database/pagination.go:17]: I think go wants this below the imports

Comment (sethvargo) [reply pkg/database/pagination.go]: Oh yea, fixed 😄

**Review by whaught — APPROVED** [review]

**Pull request merged**